doxy

```diff
@@ -3,122 +3,52 @@
 here=`pwd`
 # echo ${here}
 
-# doxyfile="dox/DoxyHtml"
-# if [ -f ${doxyfile} ]
-# 	then
-# 		doxygen ${doxyfile}
-# fi
-
-# doxyfile="dox/DoxyMan"
-# if [ -f ${doxyfile} ]
-# 	then
-# 		doxygen ${doxyfile}
-# fi
-
+doxyfiles=("DoxyLangman" "DoxyRefman"            "DoxyUserman"      "DoxyStlman")
+locations=("langman"     "refman"                "userman"          "stlman")
+books=(    "Bill_Primer" "reference_manual"      "user_manual"      "standard_library_reference")
+names=(    "Bill_Primer" "Bill_Reference_Manual" "Bill_User_Manual" "Bill_Standard_Library_Reference")
 
+# html forward and config
+cp dox/index.html  docs/index.html
+cp dox/_config.yml docs/_config.yml
 
+# html docs
 doxyfile="dox/DoxyHome"
 if [ -f ${doxyfile} ]
```

Comment:
Consider an error if the dox file is missing.  But, this 11 lines could be the following in make:
```
.PHONY: docs-html
docs-html:
<tab>: doxygen dox/DoxyHome
```

Reply:
I still don't see an error if the main doxyfile is missing.

Reply:
This is with respect to the html.  If that file is missing (which for html should not be missing), you have a problem and you won't detect it.